scripts/functions/detect/system

```diff
@@ -21,7 +21,11 @@ __rvm_detect_system()
 
       _system_type="Linux"
 
-      if [[ -f /etc/lsb-release ]] && __rvm_detect_system_from_lsb_release
+      if [[ -f /etc/upstream-release/lsb-release ]] && __rvm_detect_system_from_lsb_release upstream
```

Comment:
I think we should implement it the other way around. First check the `lsb_release` and only when it returns `1` (fails), check the `upstream-release`.

Reply:
The problem with that is lsb_release is also present in mint, so it would enter that check returning incorrect value.

Reply:
I see your point. If `mint` follows ubuntu, maybe we should just delete `requirements/mint` and remove it from `lsb_release/__rvm_detect_system_from_lsb_release`? This way `__rvm_detect_system()` would return `1` and the proposed check on `upstream-release` would work correctly.

Reply:
Hi, sorry for late response, I was out of home. 
Mmmm, I don't know the implications of doing that, I don't know the code of rvm very well (I'm new to rvm code)